### ISSUE TITLE
Wizard API for Solidity

### ISSUE THREAD

**Issue by ericglau** [OpenZeppelin/contracts-wizard#129]
Solidity equivalent of https://github.com/OpenZeppelin/contracts-wizard/pull/127

**Comment by rg12301** [comment]
**Hello ser!**
This comment is to request you for an earlier release of **Wizard API for Solidity**. My team is participating in a hackathon for which we are building a one-click solution to deploy smart contracts. We found no better option other than OpenZeppelin Contracts as they are secure and well audited. If my request could not be fulfilled in a week or two, please guide me with a workaround for the same.
**Also, accept my gratitude for your awesome work.**

**Comment by ericglau** [comment]
@rg12301 Thanks for the input. We'll aim to have something available for you to use within the next couple of weeks.

**Comment by rg12301** [comment]
Awesome!! thank you for considering my request.

**Comment by ericglau** [comment]
@rg12301 The API would look like [this](https://github.com/ericglau/contracts-wizard/blob/solidityapi/packages/core/README.md), let me know if you have any feedback on it.

You could build the project from [this source](https://github.com/ericglau/contracts-wizard/tree/solidityapi) if you would like to try it out in the meantime before it is released.

**Comment by rg12301** [comment]
Cool! I'll try it and let you know.

**Comment by rg12301** [comment]
@ericglau I was able to build and print smart contracts using the API. However, I got stuck when tried to deploy the application to Vercel. Deployment failed due to the following error:

![image](https://user-images.githubusercontent.com/56386618/173533296-52b01658-1b38-4a6e-b8a4-ddc49facb330.png)

As in the above ss, this issue is in the `packages/core/src/contract.ts:92:16`. I also tried after modifying the `tsconfig.json` as described, but the error continues to appear.
